homeassistant/components/mill/climate.py

```diff
@@ -41,20 +43,20 @@
 async def async_setup_entry(hass, entry, async_add_entities):
     """Set up the Mill climate."""
 
-    mill_data_connection = hass.data[DOMAIN]
+    mill_data_coordinator = hass.data[DOMAIN]
 
     dev = []
-    for heater in mill_data_connection.heaters.values():
-        dev.append(MillHeater(heater, mill_data_connection))
-    async_add_entities(dev)
+    for heater in mill_data_coordinator.data.values():
+        dev.append(MillHeater(mill_data_coordinator, heater))
+    async_add_entities(dev, True)
```

Comment:
You shouldn't have to trigger an update when adding entities since the data coordinator already contains the latest data.